engine/src/flutter/lib/ui/painting.dart

```diff
@@ -195,28 +194,16 @@ class Color {
       g = (g & 0xff) / 255,
       b = (b & 0xff) / 255;
 
-  /// The normalized alpha channel of this color.
-  ///
-  /// A value of `0.0` means this color is fully transparent. A value of `1.0`
-  /// means this color is fully opaque.
+  /// The alpha channel of this color.
```

Comment:
I think it was more clear before, if incorrect. I think we should provide some guidance on what each of a, r, and g mean and what are their bounds. Maybe something like: 

```
For example, in the sRGB color space, where 0 is the maximum value and 255 is the minimum, a would be a value between 0 and 1.
```